Candidate: CVE-2012-2135
PublicDate: 2012-04-25
References:
 http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2012-2135
Description:
 In the utf-16 decoder after calling unicode_decode_call_errorhandler
 aligned_end is not updated. This may potentially cause data leaks,
 memory damage, and crash. The bug introduced by implementation of the
 issue #4868. In a similar situation in the utf-8 decoder aligned_end
 is updated.
Ubuntu-Description:
Notes:
 jdstrand> python3 only
 jdstrand> patch in upstream bug is in Debian, but not committed upstream
Bugs:
 http://bugs.python.org/issue14579
 http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=670389
Priority: low
Discovered-by: Serhiy Storchaka
Assigned-to: jdstrand

Patches_python3.2:
 patch: http://bugs.python.org/file25366/utf16_error_handling-3.2_4.patch
upstream_python3.2: needs-triage
hardy_python3.2: DNE
lucid_python3.2: DNE
natty_python3.2: needed
oneiric_python3.2: needed
precise_python3.2: needed
devel_python3.2: needed

Patches_python3.3:
upstream_python3.3: needs-triage
hardy_python3.3: DNE
lucid_python3.3: DNE
natty_python3.3: DNE
oneiric_python3.3: DNE
precise_python3.3: DNE
devel_python3.3: needs-triage

Patches_python3.1:
upstream_python3.1: needs-triage
hardy_python3.1: DNE
lucid_python3.1: needed
natty_python3.1: needed
oneiric_python3.1: DNE
precise_python3.1: DNE
devel_python3.1: DNE
